### DISCUSSION TITLE
Enable experimental features in package config, such as `Advanced Perceptual Contrast Algorithm (APCA)`

### DISCUSSION THREAD

**Discussion by vincerubinetti** [GoogleChrome/lighthouse#16237] [Q&A] [answered]
I'm trying to use Lighthouse as a package for automated performance and accessibility testing for my app.

Is there any way to enable APCA (and other experimental things that are in the Dev Tools version of Lighthouse) when using Lighthouse as a package?

<img width="778" alt="Screenshot 2024-10-28 at 7 21 56 PM" src="https://github.com/user-attachments/assets/2c95d685-5af4-4d8d-9201-4f07ff76dc6d">

**Comment by adamraine** [comment]
Our color contrast advice comes from axe-core, please see https://github.com/dequelabs/axe-core/issues/3325

**Reply by vincerubinetti** [reply]
Yes I'm aware of that issue. It's about adding an APCA check to axe-core. But Lighthouse in Chrome Dev Tools does support APCA. How? Where does that support come from? Does the functionality that's in Chrome Dev Tools not perfectly overlap with the functionality of this package?

**Reply by vincerubinetti** [reply, marked as answer]
> Does the functionality that's in Chrome Dev Tools not perfectly overlap with the functionality of this package?

Okay, the answer to this is yes; the functionalities are different. If this is actually the code for the Chrome Dev Tools panel, it seems like the APCA calculation is done just in the frontend of the dev tools panel, and not in the standalone Lighthouse package:

https://github.com/search?q=repo%3AChromeDevTools%2Fdevtools-frontend%20apca&type=code
